sdk/keyvault/azure.keyvault.secrets.txt

```diff
@@ -0,0 +1,57 @@
+**** Package/Module azure.keyvault.secrets****
+class SecretClient(KeyVaultClientBase, object):
+	vault_url:str
+	credential
+
+	vault_url
+
+	def __init__(self, vault_url: str, credential, transport=None, api_version=None, **kwargs): ...
```

Comment:
`transport` and `api_version` should be `**kwargs`.

Reply:
#7772